MiniApp/Classes/JavascriptBridge/MiniAppScriptMessageHandler.swift

```diff
@@ -49,6 +52,36 @@ internal class MiniAppScriptMessageHandler: NSObject, WKScriptMessageHandler {
         executeJavaScriptCallback(responseStatus: .onSuccess, messageId: messageId, response: uniqueId)
     }
 
+    func requestPermission(requestParam: RequestParameters?, callbackId: String) {
+        guard let requestParamValue = requestParam?.permission else {
+            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: MiniAppJavaScriptError.invalidPermissionType.rawValue)
+            return
+        }
+        let requestPermissionType = MiniAppPermissionType(rawValue: requestParamValue)
+
+        switch requestPermissionType {
+        case .location:
+            getPermissionResult(requestParam: requestParamValue, callbackId: callbackId)
+        default:
+            executeJavaScriptCallback(responseStatus: .onError, messageId: callbackId, response: MiniAppJavaScriptError.invalidPermissionType.rawValue)
+        }
+    }
+
+    func getPermissionResult(requestParam: String, callbackId: String) {
```

Comment:
Shouldn't this function accept `MiniAppPermissionType` instead of `requestParam`?

The `requestParam` here is just being used to create the `MiniAppPermissionType`, but this was already created above in `requestPermission` (as `requestPermissionType `) so you could just pass that to this function.

Reply:
Done 👍